src/error.rs

```diff
@@ -22,8 +22,10 @@ impl Error {
         let message: HString = message.into();
 
         // RoOriginateError creates the error object and associates it with the thread.
+        // Need to ignore the result, as that is the delay-load error, which would mean
+        // that there's no WinRT to tell about the error.
```

Comment:
Technically we need to create the `IRestrictedErrorInfo` that `RoOriginateError` would have created ourselves as a fallback, but Rust/WinRT doesn't yet support that so this is sufficient for now.